packages/models-library/src/models_library/clusters.py

```diff
@@ -106,11 +108,14 @@ class Config:
         use_enum_values = True
 
     def to_clusters_db(self, only_update: bool) -> Dict[str, Any]:
-        db_model = self.dict(
-            by_alias=True,
-            exclude={"id", "access_rights"},
-            exclude_unset=only_update,
-            exclude_none=only_update,
+        db_model = json.loads(
+            self.json(
+                by_alias=True,
+                exclude={"id", "access_rights"},
+                exclude_unset=only_update,
+                exclude_none=only_update,
+                encoder=create_json_encoder_wo_secrets(BaseCluster),
```

Comment:
so I would be interested on how to do it, because I need to be able to encrypt/unencrypt. and this seems to go against the rules... I need that password to connect with the dask gateway.